frontend/packages/helm-plugin/src/components/details-page/resources/HelmReleaseResourcesRow.tsx

```diff
@@ -37,16 +37,16 @@ const HelmReleaseResourcesRow: React.FC<RowFunctionArgs<K8sResourceKind>> = ({ o
       <TableData className={tableColumnClasses.name}>
         <ResourceLink
           kind={kind}
-          name={resource.metadata.name}
-          namespace={resource.metadata.namespace}
+          name={resource.metadata?.name}
+          namespace={resource.metadata?.namespace}
         />
       </TableData>
       <TableData className={tableColumnClasses.type}>{resource.kind}</TableData>
       <TableData className={tableColumnClasses.status}>
         <HelmReleaseResourceStatus resource={resource} />
       </TableData>
       <TableData className={tableColumnClasses.created}>
-        <Timestamp timestamp={resource.metadata.creationTimestamp} />
+        <Timestamp timestamp={resource.metadata?.creationTimestamp || ''} />
```

Comment:
maybe we can copy the timestamp changes from your other PR so we don't need the empty string fallback here?